ledger/ledger-on-memory/src/app/scala/com/daml/ledger/on/memory/ExtraConfig.scala

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2020 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.on.memory
+
+final case class ExtraConfig(
+    maxInboundMessageSize: Int
+)
+
+object ExtraConfig {
+  val defaultMaxInboundMessageSize: Int = 64 * 1024 * 1024
```

Comment:
Seems high. I'm all for having this configurable but perhaps we can start with 4 MB?

Reply:
Done